Common/Sources/Extensions/UIViewController.swift

```diff
@@ -31,6 +31,14 @@ public extension UIViewController {
         if #available(iOS 11.0, *) {
             self.navigationController?.navigationBar.prefersLargeTitles = true
             currentNavigationItem.largeTitleDisplayMode = mode
+            if #available(iOS 13.0, *) {
+                ThemeManager.applyToUINavigationBar()
+            } else {
+              self.navigationController?.navigationBar.largeTitleTextAttributes = [
+                NSAttributedString.Key.foregroundColor: Theme.NavigationBar.largeTitleColor,
+                NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 18.0)
+              ]
+            }
```

Comment:
@nanbehera there's a reason why `ThemeManager.applyToUINavigationBar()` is optional because not every integrator wants our custom navigation bar implementation. So you have to remove this logic from here

Reply:
Ok . Code reverted back